core/src/main/java/com/scalar/db/io/BigIntColumn.java

```diff
@@ -33,9 +31,9 @@ private BigIntColumn(String name) {
 
   private BigIntColumn(String name, long value, boolean hasNullValue) {
     this.name = Objects.requireNonNull(name);
-    checkArgument(
-        value >= MIN_VALUE && value <= MAX_VALUE,
-        "Out of range column value for BigInt. value: " + value);
```

Comment:
This string building always happened in spite of `IllegalArgumentException` thrown or not.